processing/src/main/java/org/apache/druid/segment/filter/OrFilter.java

```diff
@@ -99,6 +99,17 @@ public boolean canVectorizeMatcher()
     return filters.stream().allMatch(Filter::canVectorizeMatcher);
   }
 
+  @Override
+  public boolean shouldUseIndex(BitmapIndexSelector bitmapIndexSelector)
```

Comment:
Looks like this impl of `shouldUseIndex()` and `supportsBitmapIndex()` could be moved up to `BooleanFilter` and `AndFilter` could also use them.

Reply:
Also `supportsSelectivityEstimation`